shell/platform/embedder/embedder.h

```diff
@@ -502,11 +544,6 @@ typedef struct {
   double y;
 } FlutterPoint;
 
```

Comment:
Nit: maybe move FlutterRect, FlutterPoint, and FLutterRoundedRect up as well so these similar structs all remain together.